src/core/display.cpp

```diff
@@ -820,6 +819,218 @@ bool showJpeg(FS fs, String filename, int x, int y) {
 }
 
 
+//####################################################################################################
+// Draw a GIF on the TFT
+// derived from https://github.com/bitbank2/AnimatedGIF/blob/master/examples/TFT_eSPI_memory/TFT_eSPI_memory.ino
+// and https://github.com/bitbank2/AnimatedGIF/blob/master/examples/best_practices_example/best_practices_example.ino
+//####################################################################################################
+
+#include <AnimatedGIF.h>
+
+#define NORMAL_SPEED
+//#define USE_DMA
+
+#ifdef USE_DMA
+  uint16_t usTemp[2][SAFE_STACK_BUFFER_SIZE]; // Global to support DMA use
+#else
+  uint16_t usTemp[1][SAFE_STACK_BUFFER_SIZE];    // Global to support DMA use
+#endif
```

Comment:
These are the villains of DRAM memory..

Isn't it possible to use malloc to address it into heap memory, and use the funtions to point into this address?

Reply:
just tried , still seem to overflow dram0_0_seg...